websites/G/Gartic Phone/presence.ts

```diff
@@ -2,42 +2,45 @@ const presence = new Presence({
     clientId: "803366782722244638"
 });
 
-  const browsingStamp = Math.floor(Date.now() / 1000);
-  
-  presence.on("UpdateData", async () => {
-    const presenceData: PresenceData = {
-      largeImageKey: "glogo"
-    };
-  
-    if (window.location.hostname.includes("garticphone")) {
-      if (window.location.pathname.endsWith('lobby')){
-        presenceData.startTimestamp = browsingStamp;
-        presenceData.details = "Browsing lobby";
-      } 
-       else if (window.location.pathname.endsWith('start')){
-        presenceData.startTimestamp = browsingStamp;
-        presenceData.details = "Start a story";
-      }
-      else if (window.location.pathname.endsWith('draw')){
-        presenceData.startTimestamp = browsingStamp;
-        presenceData.details = "Drawing...";
-      }
-      else if (window.location.pathname.endsWith('write')){
-        presenceData.startTimestamp = browsingStamp;
-        presenceData.details = "Writing...";
-      }
-      else if (window.location.pathname.endsWith('book')){
-        presenceData.startTimestamp = browsingStamp;
-        presenceData.details = "Look at the album";
-      } else {
-        presenceData.startTimestamp = browsingStamp;
-        presenceData.details = "Browsing the home page";
-      }
+var browsingStamp = Math.floor(Date.now() / 1000);
+
+presence.on("UpdateData", async () => {
+  const presenceData: PresenceData = {
+    largeImageKey: "glogo"
+  };
+
+  if (window.location.hostname.includes("garticphone")) {
+    if (window.location.pathname.endsWith('lobby')){
+      presenceData.startTimestamp = browsingStamp;
+      presenceData.details = "Awaiting the launch of the game..."
+    } 
+    else if (window.location.pathname.endsWith('start')){
+      presenceData.startTimestamp = browsingStamp;
+      presenceData.details = "Start a story";
+    }
+    else if (window.location.pathname.endsWith('draw')){
+      presenceData.startTimestamp = browsingStamp;
+      presenceData.details = `Drawing:`;
+      presenceData.state = `${document.querySelector(".jsx-1934821697 h3").textContent}`
```

Comment:
I think this shouldn't be displayed (or give an option to disable it), because if you play with other people on discord they're not supposed to see what you were asked to draw

Reply:
Thank you !
![image](https://user-images.githubusercontent.com/36775882/105917892-532d2c00-6033-11eb-8e4d-e303cff6758c.png)